docs/index.rst

```diff
@@ -3,16 +3,31 @@ Qiskit Terra documentation
 ##########################
 
 .. toctree::
-  :maxdepth: 2
-  :hidden:
+   :hidden:
 
-  How-to Guides <how_to/index>
-  API References <apidocs/terra>
-  Explanation <explanation/index>
-  Migration Guides <migration_guides/index>
-  Release Notes <release_notes>
+   Documentation Home <self>
+   qc_intro
+   getting_started
+   intro_tutorial1
+   How-to Guides <how_to/index>
+   API Reference <apidoc/terra>
+   Explanation <explanation/index>Migration Guides <migration_guides/index>
+   release_notes
+   configuration
+   GitHub <https://github.com/Qiskit/qiskit-terra>
+   faq
```

Comment:
The formatting's a bit broken here, and the "API reference" link doesn't point correctly, I think.